taxi/views.py

```diff
@@ -87,3 +88,37 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/driver_form.html"
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+@login_required
+def assign_driver(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    if request.user not in car.drivers.all():
+        car.drivers.add(request.user)
+    return redirect(reverse("taxi:car-detail", kwargs={"pk": pk}))
+
+
+@login_required
+def remove_driver(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    if request.user in car.drivers.all():
+        car.drivers.remove(request.user)
+    return redirect(reverse("taxi:car-detail", kwargs={"pk": pk}))
```

Comment:
The `remove_driver` function currently allows any logged-in user to remove themselves from a car. Consider adding permission checks to ensure that only authorized users can perform this action.